tests/endtoend_tests/test_endtoend.py

```diff
@@ -113,6 +114,48 @@ def test_build_generate_only(dummy_tesseract_location):
     assert dockerfile_path.exists()
 
 
+@pytest.mark.parametrize("skip_checks", [True, False])
+def test_build_skip_checks(
+    docker_cleanup, dummy_tesseract_location, tmp_path, skip_checks
+):
```

Comment:
Can we do this in a way that doesn't introduce a (fairly expensive) E2E test for this small feature?

For example, generate a build context with `--generate-only`, then verify that `tesseract-runtime check` isn't in the generated Dockerfile.